docs/src/app/(public)/(content)/react/components/alert-dialog/demos/detached-triggers-controlled/css-modules/index.tsx

```diff
@@ -1,6 +1,7 @@
+'use client';
 import * as React from 'react';
 import { AlertDialog } from '@base-ui-components/react/alert-dialog';
-import styles from '../../index.module.css';
+import styles from '../../_index.module.css';
```

Comment:
Nitpick: Was there any specific reason for the file rename?
Would `index.module.css` be mistreated as the index file? 🤔

Reply:
The filenames are normalized when displayed, so `button/index.tsx` would become `button.tsx`. If you want `index.tsx` to be shown, it should be escaped in the filesystem with `_index`